Services/Administration/OJS.Services.Administration.Data/Implementations/ContestsDataService.cs

```diff
@@ -139,11 +139,10 @@ public Task<bool> IsUserInExamGroupByContestAndUser(int id, string? userId)
 
         private async Task<int> GetMaxPointsByIdAndProblemGroupsFilter(int id, Expression<Func<ProblemGroup, bool>> filter)
             => await this.GetByIdQuery(id)
-                .Select(c => c.ProblemGroups
-                    .AsQueryable()
-                    .Where(pg => pg.Problems.Any(p => !p.IsDeleted))
-                    .Where(filter)
-                    .Sum(pg => (int?)pg.Problems.First().MaximumPoints))
+                .SelectMany(c => c.ProblemGroups)
+                .Where(pg => pg.Problems.Any(p => !p.IsDeleted))
+                .Where(filter)
+                .Select(pg => (int?)pg.Problems.First().MaximumPoints)
```

Comment:
Where is the `Sum`. It seems you have modified the logic, or I'm missing something?

Reply:
Fixed